chicagobub/scripts/bubble_chart.js

```diff
@@ -61,44 +68,63 @@ function bubbleChart() {
         value: +d.AF2LE001,
         name: d.TRACTA,
         group: d.THREECITY,
-        year: d.INC1970,
+        1970: d.INC1970,
+        1980: d.INC1980,
+        1990: d.INC1990,
+        2000: d.INC2000,
+        2010: d.INC2010,
+        2016: d.INC2016,
```

Comment:
Here is the other big change, we have a different data value for each bubble, for each year. This allows us to reach into each bubble in `nodeYearPos()` to access the right value.